src/shipit_static_analysis/shipit_static_analysis/lint.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, repo_dir, repo_path, column, level, lineno, linter, message,
         self.path = repo_path
         self.column = column
         self.level = level
-        self.line = lineno
+        self.line = int(lineno)  # mozlint sometimes produce strings here
```

Comment:
@ahal This was with Codespell.
Do you think we could enforce that at the mozlint level?